docker-compose.vpn.yml

```diff
@@ -0,0 +1,132 @@
+version: '2.4'
+
+services:
+
+  # main application
+  nefarious:
+    labels:
+    - com.centurylinklabs.watchtower.enable=true
+    image: lardbit/nefarious
+    restart: always
+    ports:
+    - "8000:80"
+    environment:
+      # https://github.com/kennethreitz/dj-database-url
+      DATABASE_URL: "sqlite:////nefarious-db/db.sqlite3"
+      REDIS_HOST: "redis"
+      NEFARIOUS_USER: ${NEFARIOUS_USER:-admin}
+      NEFARIOUS_PASS: ${NEFARIOUS_PASS:-admin}
+      HOST_DOWNLOAD_PATH: ${HOST_DOWNLOAD_PATH:-/tmp/}
+    volumes:
+      # persistent named volume for sqlite database
+      - nefarious-db:/nefarious-db
+    logging:
+      options:
+        max-size: 10m
+    depends_on:
+    - redis
+    - jackett
+    - celery
+
+  # background task queue
+  celery:
+    labels:
+      - com.centurylinklabs.watchtower.enable=true
+    image: lardbit/nefarious
+    restart: always
+    entrypoint: /env/bin/celery -A nefarious worker --concurrency ${NUM_CELERY_WORKERS:-0} --beat --loglevel=INFO
+    environment:
+      # https://github.com/kennethreitz/dj-database-url
+      DATABASE_URL: "sqlite:////nefarious-db/db.sqlite3"
+      REDIS_HOST: "redis"
+      HOST_DOWNLOAD_PATH: ${HOST_DOWNLOAD_PATH:-/tmp}
+      INTERNAL_DOWNLOAD_PATH: /downloads/completed
+    volumes:
+      - ${HOST_DOWNLOAD_PATH:-/tmp}:/downloads
+      # persistent named volume for sqlite database
+      - nefarious-db:/nefarious-db
+    logging:
+      options:
+        max-size: 10m
+    depends_on:
+      - redis
+
+  # in-memory database for task queue
+  redis:
+    image: redis
+    restart: always
+    mem_limit: 200m
+    # TODO - use "expose" vs "ports" to limit external access when not in development
+    ports:
+    - "6379:6379"
+
+  # torrent indexer service
+  # See Documentation for additonal environment options that may be needed per
+  # your vpn provider
+  # VPN Docs: https://haugene.github.io/docker-transmission-openvpn/
+  jackett:
+    labels:
+      - com.centurylinklabs.watchtower.enable=true
+    image: linuxserver/jackett
+    mem_limit: 200m
+    restart: always
+    network_mode: service:transmission
```

Comment:
@lardbit this is the part that I missed that ties it together to route over the transmission container.